app/Config/core.php

```diff
@@ -149,13 +149,13 @@
  *	`manager_index()` and `/manager/controller/index`
  *
  */
-	//Configure::write('Routing.prefixes', array('admin'));
+    //Configure::write('Routing.prefixes', array('admin'));
 
 /**
  * Turn off all caching application-wide.
  *
  */
-	//Configure::write('Cache.disable', true);
+    //Configure::write('Cache.disable', true);
```

Comment:
__[staging][code_sniffer]__
* No space found before comment text; expected "// Configure::write('Cache.disable', true);" but found "//Configure::write('Cache.disable', true);"
* Inline comments must end in full-stops, exclamation marks, or question marks
* There must be no blank line following an inline comment